pkg/intercp/catalog/catalog.go

```diff
@@ -15,7 +17,7 @@ type Instance struct {
 }
 
 func (i Instance) InterCpURL() string {
-	return fmt.Sprintf("grpcs://%s:%d", i.Address, i.InterCpPort)
+	return fmt.Sprintf("grpcs://%s", net.JoinHostPort(i.Address, strconv.Itoa(int(i.InterCpPort))))
```

Comment:
Classic :) I wonder if there's a linter for this we almost systematically do this mistake!